### ISSUE TITLE
Issue apparently very similar to 7500 on Windows 10

### ISSUE THREAD

**Issue by AdamRichman** [hashicorp/packer#7672]
Megan, I am running into the same issue as described in 7500 but on Windows 10.  I pulled the latest build (1.4.1 circa 5.21.2019).  The error seems nearly identical so was not sure if it was a platform difference or something else environmental as the last comment in issue 7500 refers to it being fixed in the master.

Running with elevated rights on:
OS Name	Microsoft Windows 10 Pro
Version	10.0.17763 Build 17763

User has been added to the Hyper-V Admin Group

Log at error:
==> hyperv-iso: Enabling Integration Service...
==> hyperv-iso: PowerShell error: Hyper-V\Add-VMDvdDrive : Failed to add device 'Virtual CD/DVD Disk'.
==> hyperv-iso: Hyper-V Virtual Machine Management service Account does not have permission to open attachment.
==> hyperv-iso: 'dev-hyperv-base_name' failed to add device 'Virtual CD/DVD Disk'. (Virtual machine ID 2EA8890E-AF47-4C06-A9F4-1199E677B42F)
==> hyperv-iso: 'dev-hyperv-base_name': Hyper-V Virtual Machine Management service account does not have permission required to open attachment
==> hyperv-iso: 'C:\Users\xxxx\repos\packer_builds\packer_cache\08478213f4bb76a558776915c085b9de13744f87.iso'. Error: 'General access denied error' (0x80070005). (Virtual
==> hyperv-iso: machine ID 2EA8890E-AF47-4C06-A9F4-1199E677B42F)
==> hyperv-iso: At C:\Users\xxxx\AppData\Local\Temp\powershell294707329.ps1:3 char:18
==> hyperv-iso: + ... ontroller = Hyper-V\Add-VMDvdDrive -VMName $vmName -path $isoPath -Pa ...
==> hyperv-iso: +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
==> hyperv-iso:     + CategoryInfo          : PermissionDenied: (:) [Add-VMDvdDrive], VirtualizationException
==> hyperv-iso:     + FullyQualifiedErrorId : AccessDenied,Microsoft.HyperV.PowerShell.Commands.AddVMDvdDrive
==> hyperv-iso: Unregistering and deleting virtual machine...
==> hyperv-iso: Deleting output directory...
==> hyperv-iso: Deleting build directory...
Build 'hyperv-iso' errored: PowerShell error: Hyper-V\Add-VMDvdDrive : Failed to add device 'Virtual CD/DVD Disk'.

Reference:  
I was finally able to create a repro case for this.  It turns out that it's already fixed on the master branch, probably by the PR Adrien linked above.  If you're in a big rush for a fix you can use the Packer [nightly](https://github.com/hashicorp/packer/releases/tag/nightly) build until we release 1.4.1 tomorrow-ish :)

_Originally posted by @SwampDragons in https://github.com/hashicorp/packer/issues/7500#issuecomment-492389793_

**Comment by SwampDragons** [comment]
Shoot. I'm hoping this is an issue where you're still stuck in a bad state after using v1.4.0.  

When I managed to reproduce in 7500, it was by:

- downloading an iso file to my desktop
- clearing out all symlinks from my packer_cache
- clearing out files from my packer_cache with the same checksum of that iso
- running a v1.4.0 build using the path to the local iso as my iso_url

The error was coming from the fact that in v1.4.0 we created a symlink in the packer_cache to the file's location, rather than copying or moving the file into the cache. This symlink was causing the permission errors. If I had to guess, I'd put money on your packer_cache still having this bad symlink.  What happens if you clear your cache, or at least remove all links from your cache?
